src/ProjectTask.php

```diff
@@ -143,6 +143,25 @@ public function canUpdateItem(): bool
         return false;
     }
 
+    public static function getMyTasksURL(bool $full)
+    {
+        return self::getSearchURL($full) . '?' . Toolbox::append_params([
+            'criteria' => [
+                [
+                    'field' => 87,
+                    'searchtype' => 'equals',
+                    'value' => 'myself'
+                ],
+                [
+                    'link' => 'AND',
```

Comment:
Shouldn't it be `OR` ? 

```suggestion
                    'link' => 'OR',
```